frontend/catalyst/utils/wrapper.cpp

```diff
@@ -13,16 +13,15 @@
 // limitations under the License.
 
 #include <csignal>
-#include <iostream>
-#include <pybind11/pybind11.h>
+#include <nanobind/nanobind.h>
 
 // TODO: Periodically check and increment version.
 // https://endoflife.date/numpy
 #define NPY_NO_DEPRECATED_API NPY_1_24_API_VERSION
 
-#include "numpy/arrayobject.h"
+#include "numpy/ndarrayobject.h"
```

Comment:
Just curious about this change

Reply:
This was flagged by my [clangd `#include` manager](https://clangd.llvm.org/guides/include-cleaner), which

> follows the [include-what-you-use](https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md) model: each source file should #include headers that declare the symbols it references, and no others.

[`numpy/arrayobject.h`](https://github.com/numpy/numpy/blob/v2.0.1/numpy/_core/include/numpy/arrayobject.h) appears to be a sort of alias for `"ndarrayobject.h"` in numpy 2.x. In numpy 1.x, [`numpy/arrayobject.h`](https://github.com/numpy/numpy/blob/v1.26.5/numpy/core/include/numpy/arrayobject.h) also includes `"npy_interrupt.h"`; we don't use the symbols defined in this header which is why the clangd tool complained.

I also found in the NumPy documentation that including the `numpy/ndarrayobject.h` header is preferred: https://numpy.org/devdocs/reference/c-api/array.html#including-and-importing-the-c-api

Reply:
Oh neat, I've looking for C++ include managers